domain/src/main/scala/io/github/positionpal/message/ChatMessageADT.scala

```diff
@@ -0,0 +1,33 @@
+package io.github.positionpal.message
+
+
+object ChatMessageADT:
+
+  type ChatMessage = ChatMessageImpl
+
+  /**
+   * Represents the data that a ChatMessage exposes
+   */
+  trait MessageOps:
+    def text: String
+    def timestamp: String
+    def from: String            // TODO: Change return type to User
+    def to: String              // TODO: Change return type to Group
```

Comment:
todos comments, I don't know whether they are intentional or have been forgotten

Reply:
Are intentional, the actual domain object will be added in the next feature release.

Reply:
Thank you for the catch on the QA plugin, It was removed by mistake.